packages/inferno-from-react-generator/src/__tests__/generator.test.ts

```diff
@@ -70,4 +70,8 @@ describe('generator', () => {
     const generator = await createTestGenerator('types', createGenerator());
     expect(`${generator[0].code}`).toMatchSnapshot();
   });
```

Comment:
Let's add blank lines between test cases:

```suggestion
  });
  
```